src/redux/actions/resetPassword.js

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+import actions from './index';
+
+const {
+  RESET_PASSWORD_SUCCESS, RESET_PASSWORD_FAILURE,
+  UPDATE_PASSWORD_FAILURE, UPDATE_PASSWORD_SUCCESS,
+} = actions;
+const BASE_URL_1 = 'https://fargo-ah-staging.herokuapp.com/api';
```

Comment:
well done, I think handling this base url in a dynamic way is just appropriate, what about using an environment variable for this?

Reply:
Thank you Sinmi for your suggestion, while it is a good option that the team is still trying to find a way around, I suggest we add the base URL to axios using `axios.defaults.baseURL`